README.md

```diff
@@ -135,7 +144,49 @@ Good job! You have now generated the skeleton for your package:
 ```
 
 For an explanation of what's there, read on in the [project_setup.md](template/project_setup.md) file.
-There are also instructions on how to [apply the template to an existing Python package](ADD_TO_EXISTING_PACKAGE.md).
+
+### Option 2: Apply to pre-existing code
+
+To apply the template to pre-existing code, you can use the same `copier copy`
+command as when creating a new package, except that you point to the folder
+containing your existing code rather than a new one:
+
+```shell
+copier copy https://github.com/nlesc/python-template.git path/to/existing/code
+```
+
+This works because if `path/to/destination` already exists, `copier` will
+update what is already there by either adding new files or updating
+existing files. Copier will ask to overwrite any files that resulted in
+conflicts. Especially if your files are already under version control, it is
+recommended to answer 'yes' for all files, you will still be able to review
+the changes suggested by the template.
+
+### Option 3. Updating a template-made package
+
+Copier provides the functionality for re-applying the template to a previously
+created project using the `copier update` command. This has two effects:
+
+1. Your project will be updated according to the latest version of the template
+2. You can change any of your previous answers to apply these changes
+   throughout your entire project.
```

Comment:
Please, never manually update answers in `.copier-answers.yml` as it will result in unexpected behavior.

Reply:
I've added a `> [!CAUTION]` block with this information